hypertrace-core-graphql-common-schema/src/main/java/org/hypertrace/core/graphql/common/schema/typefunctions/UnknownScalar.java

```diff
@@ -67,6 +69,9 @@ public Object parseLiteral(Object input) throws CoercingParseLiteralException {
                     return ((ArrayValue) input)
                         .getValues().stream().map(recurse).collect(Collectors.toUnmodifiableList());
                   }
+                  if (input instanceof EnumValue) {
+                    return ((EnumValue) input).getName();
```

Comment:
Hmm - so we still need enough context to know what type of enum this should deser into right? And that'll be the job of the arg deserializer/request building phase? I guess it should work in the concrete filter application though - we can give it a shot.

Reply:
> Hmm - so we still need enough context to know what type of enum this should deser into right? And that'll be the job of the arg deserializer/request building phase?

Yes, this returns the enum name as String. So, it needs to be deserialized using `<EnumType>.valueOf(..)`